File: ios/SupportedOrientationsViewController.m
//
//  SupportedOrientationsViewController.m
//  AndyWarholAccessibilityProject
//
//  Created by ruben.niculcea on 3/28/19.
//  Copyright © 2019 Facebook. All rights reserved.
//

#import "SupportedOrientationsViewController.h"

@interface SupportedOrientationsViewController ()

@end

@implementation SupportedOrientationsViewController

- (BOOL)shouldAutorotate {
  return true;
}

- (UIInterfaceOrientationMask)supportedInterfaceOrientations {
  return UIInterfaceOrientationMaskAll;
}

@end

File: ios/SupportedOrientationsViewController.swift
//
//  SupportedOrientationsViewController.swift
//  AndyWarholAccessibilityProject
//
//  Created by Ruben Niculcea on 6/19/17.
//  Copyright © 2017 Facebook. All rights reserved.
//

import UIKit

class SupportedOrientationsViewController: UIViewController {
  override var shouldAutorotate: Bool {
    return true
  }
  
  override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
    return .all
  }
}
